assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/vision/detection_predict.py

```diff
@@ -164,8 +119,8 @@ def load_context(self, context: mlflow.pyfunc.PythonModelContext) -> None:
 
         if self._task_type in [Tasks.MM_OBJECT_DETECTION.value, Tasks.MM_INSTANCE_SEGMENTATION.value]:
             # Install mmcv and mmdet using mim, with pip installation is not working
-            subprocess.check_call([sys.executable, "-m", "mim", "install", "mmcv-full==1.7.1"])
-            subprocess.check_call([sys.executable, "-m", "mim", "install", "mmdet==2.28.2"])
+            subprocess.check_call([sys.executable, "-m", "mim", "install", "mmcv==2.0.1"])
```

Comment:
just curious, we don't need mmcv-full anymore ?

Reply:
mmcv-full is not required for 3.x upgrade. Only mmcv and mmdet.